src/axislike.rs

```diff
@@ -194,16 +196,27 @@ impl DualAxis {
     /// This cannot be changed, but the struct can be easily manually constructed.
     pub const DEFAULT_DEADZONE: f32 = 0.1;
 
-    /// Creates a [`DualAxis`] with both `positive_low` and `negative_low` in both axes set to `threshold`.
+    /// The default shape of the deadzone used by constructor methods.
+    ///
+    /// This cannot be changed, but the struct can be easily manually constructed.
+    pub const DEFAULT_DEADZONE_SHAPE: DeadZoneShape = DeadZoneShape::Cross {
```

Comment:
Should the default shape be a cross? I think ellipse is a more common, but it's not a big deal either way.

Reply:
Let's go with a circle.